src/router/index.js

```diff
@@ -15,9 +15,59 @@ import { parseQuery, stringifyQuery } from '@/utils/url-router'
 import EmbedView from '@/views/EmbedView.vue'
 import LegacyParamsView from '@/views/LegacyParamsView.vue'
 import MapView from '@/views/MapView.vue'
+import TestEmbedView from '@/views/TestEmbedView.vue'
 
 const history = createWebHashHistory()
 
+const routes = [
+    {
+        path: '/',
+        redirect: { name: MAP_VIEW },
+    },
+    {
+        path: '/map',
+        name: MAP_VIEW,
+        component: MapView,
+        meta: {
+            requiresAppReady: true,
+        },
+    },
+    {
+        path: '/embed',
+        name: EMBED_VIEW,
+        component: EmbedView,
+        meta: {
+            requiresAppReady: true,
+        },
+    },
+    {
+        path: '/legacy',
+        name: LEGACY_PARAM_VIEW,
+        component: LegacyParamsView,
+        meta: {
+            requiresAppReady: true,
+        },
+    },
+    {
+        path: '/legacy-embed',
+        name: LEGACY_EMBED_PARAM_VIEW,
+        component: LegacyParamsView,
+        props: { embed: true },
+        meta: {
+            requiresAppReady: true,
+        },
+    },
+]
+
+if (store.getters.hasDevSiteWarning) {
+    // adding a special route to the embed test view
+    routes.push({
+        path: '/test-embed',
+        name: 'TestEmbedView',
+        component: TestEmbedView,
+    })
+}
```

Comment:
I'm not a big fan of this test view, especially also because it doesn't really do a realistic test as iframe will always be used outside of the same domain, this could result to CORS issue not being found by this. I think codepen are better for testing such think. 
Moreover this view don't work with test link.

Reply:
I thought it would be helpful to check that it was working without having to write some codepen or other helper, but it isn't as good as expected.
When self-hosting the iframe (same domain) there are some weird things happening in the background.
I was able to use this test view locally all fine, but this doesn't work when deployed on our DEV staging...

I'll remove it and copy the legacy codepen to make it point to the DEV staging (and update it subsequently as soon as we have that deployed on the test viewer)